### ISSUE TITLE
bend + scale is broken

### ISSUE THREAD

**Issue by dalcde** [hoodmane/spectralsequences#8]
The following does not produce the desired outcome:
```latex
\documentclass{article}
\usepackage{spectralsequences}

\begin{document}
\begin{sseqpage}[scale=0.3]
  \class(0, 0)
  \class(3, 3)
  \structline [bend left = 10] (0, 0)(3, 3)
\end{sseqpage}
\end{document}
```
Removing the `[scale = 0.3]` "fixes" the problem.

**Comment by hoodmane** [comment]
No idea what the heck this is. Maybe tomorrow I'll figure it out. Seems
vaguely like it's some bad interaction between \sseq@draw@addoffsetfalse
and \tikz@to@compute@relative. If that's the case, it's possible that it
was affected b my recent changes to the offset mechanism in tikz
coordinates. This would explain why I've never seen this behavior before if
it is very recent. Further suggestion that there is some relationship comes
from the fact that the nearly the same glitch appears if you say "\draw
(0,0) to[bend left=10] (3,3);".

It's kind of sad that this overlap exists between the execution path
between when drawing structlines and using tikz primitives, but this whole
package works because of a bunch of crazy hacks so I guess it's not so
surprising.

**Comment by hoodmane** [comment]
Fixed by be1fa51b6d002da0ba5288423da534e4d8b84851
